gm4_blossoming_pots/data/gm4_blossoming_pots/function/main.mcfunction

```diff
@@ -0,0 +1,5 @@
+execute as @e[type=minecraft:block_display,tag=gm4_blossoming_pots.display.decorated_pot] at @s unless block ~ ~ ~ minecraft:decorated_pot run kill @s
+execute as @e[type=minecraft:block_display,tag=gm4_blossoming_pots.display.flower_pot] at @s unless block ~ ~ ~ minecraft:flower_pot run kill @s
+execute as @e[type=minecraft:marker,tag=gm4_blossoming_pots.data.flower_pot] at @s unless block ~ ~ ~ minecraft:flower_pot run function gm4_blossoming_pots:flower/item/break with entity @s data
```

Comment:
Each flower pot marker has an associated block display. Maybe move the block display kill into a function executed by the marker.